src/bundle/Resources/public/js/scripts/helpers/table.helper.js

```diff
@@ -0,0 +1,19 @@
+(function (global, doc, eZ, $) {
+    parseCheckbox = (checkboxSelector) => {
+        doc.querySelectorAll(checkboxSelector).forEach((checkboxNode) => {
+            const parentRow = checkboxNode.closest('tr');
+            const activeClass = parentRow.classList.contains('c-table-view-item') ? 'c-table-view-item--active' : 'ez-table__row--active';
+
+            checkboxNode.addEventListener('change', (event) => {
```

Comment:
I would avoid creating anonymous function, especially in forEach, at the moment you create new function with each loop